app/scripts/controllers/mobileClients.js

```diff
@@ -33,6 +34,30 @@ angular.module('openshiftConsole')
           title: $routeParams.mobileclient
         }
       ];
+      ctrl.validUrlPattern = VALID_URL_PATTERN;
+
+      ctrl.setDmzUrl = function(dmzUrl) {
+        var mobileClient = angular.copy(ctrl.mobileClient);
+        if (!dmzUrl) {
+          dmzUrl = undefined;
+        }
+
+        if (mobileClient.spec['dmzUrl'] !== dmzUrl) {
```

Comment:
Can we move this check to `inlinedit`? `saveFunc` should only be triggered when a change happens